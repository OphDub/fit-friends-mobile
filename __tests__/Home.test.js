import 'react-native';
import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import Home from '../containers/Home/Home';

describe('Home Component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Home/>);
  });

  it('should match the snapshot', () => {
    wrapper = renderer.create(<Home/>).toJSON();

    expect(wrapper).toMatchSnapshot();
  });
});