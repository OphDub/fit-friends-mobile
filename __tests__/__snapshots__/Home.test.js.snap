// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component should match the snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Home
  </Text>
</View>
`;
